lib/tests-from-pull-request.js

```diff
@@ -28,28 +28,35 @@ function getTestsFromPR(context) {
   }
 }
 
-function getTestNames(prPatch) {
+function getTestNames(prPatch, context) {
   const testNames = [];
   const parsedPatch = prPatch.split('\n');
-
-  parsedPatch.forEach((line) => {
-    if (line.startsWith('+')) {
-      line = removePrefixAndTrim(line);
-      const testName = getTestName(line);
-
-      testName ? testNames.push(testName) : null;
-    }
-  });
-
-  return testNames;
+  return getConfig(context)
+    .then(config => {
+      parsedPatch.forEach((line) => {
+        if (line.startsWith('+')) {
+          line = removePrefixAndTrim(line);
+          const testName = getTestName(line, config);
+
+          testName ? testNames.push(testName) : null;
+        }
+      });
+    })
+    .then(() => testNames);
 }
 
 function removePrefixAndTrim(line) {
   return line.slice(1).trim();
 }
 
-function getTestName(line) {
-  const getTestNameRegex = /it\((.*),/;
+function getConfig(context) {
+  return Promise.resolve(context.config('config.yml'));
+}
+
+function getTestName(line, config) {
+  const testFunctionName = config && config.tdd1tTestFunctionName ?
+    config.tdd1tTestFunctionName : 'it';
```

Comment:
might be a good place to use [lodash's `get`](https://lodash.com/docs/4.17.11#get):

```
const testFunctionName = _.get(config, 'tdd1tTestFunctionName', 'it');
```

it tries to **get** the path given from the config object, and if not found, it returns the default value `it`